experiments/Manifest.toml

```diff
@@ -1,20 +1,28 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0"
```

Comment:
Shouldn't we update `experiments/Project.toml` too if we want the Manifest to be updated?

Reply:
Actually, looking at Charlie's PR to update ClimaAtmos to Julia v1.9 [here](https://github.com/CliMA/ClimaAtmos.jl/pull/1659/files#diff-9a9d70a0eb6f3494fd934c224bf0867c14b00d46fde8bcc4d65bb2d836898946), it looks like he only updates the Manifests (not Projects) of directories that have the Manifest checked in. In fact, it seems like he doesn't update any of the Projects. Maybe this is because specifying `julia = "1.8"` is the same as `julia = "1.8, 1.9"` (both restrict to `[1.8, 2.0)`)? I'm not sure it matters either way - it could be more clear to add the 1.9 in.

Reply:
My thought was that if someone clones the repo and the manifest in experiments was made with 1.9, it will fail if they try and run it with 1.8. so perhaps we should restrict the experiments compat to 1.9? Or else they need to delete it and re-instaniate. Not sure how "out of the box" it needs to run.

But sounds like what you and @AlexisRenchon suggested with respect to the Project.tomls is to leave at just "1.8" because that indicates 1.9 is also supported. I havent tested with 1.7 and Im sure it might fail at some point on an earlier Julia version.. so perhaps leaving the 1.8 there is good since we dont know if we support previous versions?

Reply:
(also, our CI will just run with 1.9)